### PR TITLE
[Kotlin Migration] CardTemplatePreviewer

### PR DESCRIPTION
**Pull request by Akshay0701**

### CODE REVIEW

**Comment by Akshay0701** [comment]
did 3 PR's on the card template activity, if any things break ping me directly XD

File: AnkiDroid/src/main/java/com/ichi2/anki/CardTemplatePreviewer.kt [hunk 1]
```diff
@@ -14,476 +14,390 @@
...
     }
 
-
-    @Override
-    protected void setTitle() {
-        if (getSupportActionBar() != null) {
-            getSupportActionBar().setTitle(R.string.preview_title);
+    override fun setTitle() {
+        if (supportActionBar != null) {
+            supportActionBar!!.setTitle(R.string.preview_title)
```
Comment (david-allison-1) [inline AnkiDroid/src/main/java/com/ichi2/anki/CardTemplatePreviewer.kt:137]: Might as well: `supportActionBar?.setTitle`

**Review by david-allison-1 — APPROVED** [review]
LGTM

**Review by mikehardy — APPROVED** [review]
approved pending that one change from `!!` to `?` - just tag me when pushed and I'll re-test all the CI build targets and merge, thanks @Akshay0701

**Comment by Akshay0701** [comment]
@mikehardy done, wanting for CI...

**Pull request merged**